src/Sylius/Bundle/ApiBundle/Serializer/CommandOperationNormalizer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Serializer;
+
+use Symfony\Component\Serializer\Exception\MissingConstructorArgumentsException;
+use Symfony\Component\Serializer\Normalizer\ContextAwareNormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class CommandOperationNormalizer implements ContextAwareNormalizerInterface
+{
+    private const ALREADY_CALLED = 'command_operation_normalizer_already_called';
```

Comment:
After your suggestion for the name:
```suggestion
final class CommandNormalizer implements ContextAwareNormalizerInterface
{
    private const ALREADY_CALLED = 'command_normalizer_already_called';
```